lib/internal/modules/cjs/loader.js

```diff
@@ -840,20 +845,22 @@ const CircularRequirePrototypeWarningProxy = new Proxy({}, {
   }
 });
 
-// Object.prototype and ObjectProtoype refer to our 'primordials' versions
+// Object.prototype and ObjectPrototype refer to our 'primordials' versions
```

Comment:
I think `ObjectPrototype` actually is the same as `global.Object.prototype`

Reply:
I guess that is somewhat out of scope of this PR though?